galaxy_ng/app/settings.py

```diff
@@ -82,3 +82,20 @@
 # ----------------------
 X_PULP_CONTENT_HOST = "localhost"
 X_PULP_CONTENT_PORT = 24816
+
+# works
+# SPECTACULAR_SETTINGS__TITLE = "Automation Hub API __TITLE"
```

Comment:
are these lines something that can be removed since title is below?

Reply:
Yeah, that is meant as an example of how to set drf_spectular settings vis env variables / dynaconf. I'll rephrase to make that clearer.